Index: target/classes/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># æ\u0096\u0087ä»¶ä¸\u008Aä¼ é\u0085\u008Dç½®\r\nspring.servlet.multipart.max-file-size=100MB\r\nspring.servlet.multipart.max-request-size=100MB\r\n# æ¨¡å\u009E\u008Bæ\u0096\u0087ä»¶ä¿\u009Då\u00AD\u0098è·¯å¾\u0084\r\nmodel.upload.path=c:/IDEAproject/ModelEvalPlat/uploads/\r\n# æ\u0095°æ\u008D®åº\u0093é\u0085\u008Dç½®\r\nserver:\r\n  port: 8002\r\n\r\nspring:\r\n  datasource:\r\n    username: root\r\n    password: 123456\r\n    url: jdbc:mysql://127.0.0.1:3306/model_eval_plat?characterEncoding=utf-8&zeroDateTimeBehavior=convertToNull\r\n    driver-class-name: com.mysql.cj.jdbc.Driver\r\n    hikari:\r\n      connection-test-query: select 1\r\n      connection-timeout: 20000\r\n      idle-timeout: 300000\r\n      maximum-pool-size: 5\r\n      minimum-idle: 5
===================================================================
diff --git a/target/classes/application.properties b/target/classes/application.properties
--- a/target/classes/application.properties	(revision 36e89b4f6740e810ed6782caa81b04407538b071)
+++ b/target/classes/application.properties	(date 1752631864712)
@@ -11,7 +11,7 @@
   datasource:
     username: root
     password: 123456
-    url: jdbc:mysql://127.0.0.1:3306/model_eval_plat?characterEncoding=utf-8&zeroDateTimeBehavior=convertToNull
+    url: jdbc:mysql://127.0.0.1:3306/modeleval?characterEncoding=utf-8&zeroDateTimeBehavior=convertToNull
     driver-class-name: com.mysql.cj.jdbc.Driver
     hikari:
       connection-test-query: select 1
